input/pagecontent/changelog.md

```diff
@@ -3,6 +3,7 @@ This is the log of changes made to the eHealth Implementation Guide.
 ## Unreleased changes
 
 ### General changes
+- Update validations for occurrence[x] in the ServiceRequest introduction.
```

Comment:
Please move to header "Resource/profile changes" and start the sentence with "Changed ....". Erik would like we use the same syntax like "Added ...", "Changed ..."